### ISSUE TITLE
undefined is not an object (evaluating 'xdate[0].toString']

### ISSUE THREAD

**Issue by fredmanxu** [wix/react-native-calendars#1679]
## Description

IOS Error: undefined is not an object (evaluating 'xdate[0].toString']

## Observed Behavior

```
import {CalendarList, WeekCalendar} from 'react-native-calendars';

render(){
  return (
             <CalendarList
                        hideDayNames={true}
                        horizontal={true}
                        pagingEnabled={true}
                        calendarWidth={WindowWidth}
                        hideExtraDays={false}
                        renderHeader={() => null}
                        theme={CALENDAR_THEME}
                        onDayPress={(day) => {
                            this.setState({selectedDay: day.dateString})
                        }}
                        markedDates={{
                            [selectedDay]: {selected: true},
                        }}
                    />
   )
}
```

## Environment

`react-native-calendars@1.1258.0`
`react-native@0.63.4`

Also specify:

1. Device&IPhone8:

## Screenshots
![1636454995(1)](https://user-images.githubusercontent.com/20528643/140910871-ab05611b-a4c5-431c-bec1-a4f9cce1c9b4.png)

**Comment by Fuyun791** [comment]
I found that the version is 1.1266.0 and it can run

**Comment by smarteye520** [comment]
Hi @fredmanxu
I got same issue yesterday and it was fixed with install another version (1.1263.0) today.
yarn add react-native-calendars@1.1263.0.
Hope that it helps you.

**Comment by fredmanxu** [comment]
> 

Thanks, It work

**Comment by fredmanxu** [comment]
> Hi @fredmanxu I got same issue yesterday and it was fixed with install another version 1.1263.0 today. _yarn add react-native-calendars@1.1263.0_ Hope that it helps you.

Thanks, It work

**Comment by khanhdevvn** [comment]
I was occus on lastest version 1.1286.0. And i must dowgrade to 1.126.0 and it has working
